vmdb/app/views/vm_or_template/explorer.html.haml

```diff
@@ -28,7 +28,7 @@
       = render(:partial => "layouts/tl_show")
       :javascript
         var miq_after_onload = "miqAsyncAjax('#{url_for(:action=>@ajax_action, :id=>@record)}');"
-    - else
+    - elsif @record.present?

```

Comment:
@romanblanco i was not able to recreate this issue. I am not sure why you had to add @record.present? here because this check has already been made on line 20, https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/views/vm_or_template/explorer.html.haml#L20

In the steps to recreate you mentioned: Clicking on Advanced settings (Virtual machines → VMs & Templates → [vm] →Advanced Settings) was causing exception, shouldn't that be vm_infra view?